lms/djangoapps/grades/models.py

```diff
@@ -172,13 +172,10 @@ def create(self, **kwargs):
             kwargs['course_id'] = kwargs['usage_key'].course_key
 
         visible_blocks_hash = VisibleBlocks.objects.hash_from_blockrecords(blocks=visible_blocks)
-        grade = self.model(
+        return super(PersistentSubsectionGradeQuerySet, self).create(
```

Comment:
While it makes sense that this change reduces queries, I don't see any tests updated to include smaller query counts. Are they missing, or am I missing them?